frontend/src/components/Histogram.jsx

```diff
@@ -29,48 +29,47 @@ export default class Histogram extends React.Component {
     console.log("Fetching case data");
 
     // if there is not an available start or end date in the data use this default ones
-    const initial_date =
-      this.props.initial_date != null ? this.props.initial_date : "2020-02-20";
-    const maximum_date =
-      this.props.maximum_date != null ? this.props.maximum_date : "2020-06-23";
+    const initDate =
+      this.props.initialDate != null ? this.props.initialDate : "2020-02-20";
+    const maximDate =
```

Comment:
Suggestion: maybe `dateInit` and `dateMaxim` would make it more obvious that these belong together?

Reply:
Done